src/main/java/ch/njol/skript/lang/VariableString.java

```diff
@@ -194,23 +186,22 @@ public static String unquote(final String s, final boolean surroundingQuotes) {
 	 * Prints errors and returns null if it is somehow invalid.
 	 * 
 	 * @param orig Unquoted string to parse.
-	 * @param mode
```

Comment:
I think this line was incorrectly removed?

Reply:
I removed it because it doesn't have a description, and my IDE gave a warning for that (a lof of the formatting stuff I took from my IDE). Should I keep parameter tags with empty descriptions?

Reply:
You can safely remove them, most of them are self-explanatory anyways but if you happen to find one that actually could make use of a description, you may as well add one for sake of completeness, it's not a requirement but a good thing for future contributors.